src/Test/StateMachine/Internal/Parallel.hs

```diff
@@ -254,6 +256,24 @@ toForkOfOps h = Fork (mkOps l) p' (mkOps r)
     dynResp = fromMaybe (error "toForkOfOps: impossible.") . fromDynamic
   mkOps _  = error "mkOps: Impossible."
 
+toBoxDrawings :: forall cmd. (IxFunctor cmd, ShowCmd cmd, HasResponse cmd) => History cmd -> Doc
```

Comment:
Would it make sense to move this (or some of it?) into the box drawing module?

Maybe align and add type signatures to help readability?